lib/server/ServerRenderer.js

```diff
@@ -72,24 +71,33 @@ function makeClientAppStartScript(clientAppRequirePath, environmentConfig, boots
         "});";
 }
 
-function makeBootstrappingLoader(scriptBody, clientAppUrl, isAsync) {
-    var loadAsync = isAsync ? "s.async = true;\n" : "";
-
+function makeBootstrappingLoader(scriptBody, clientAppUrl) {
     return "<script type=\"text/javascript\">\n" +
-        "var s = document.createElement(\"script\");\n" +
-        "var h = document.head;\n" +
-        "s.setAttribute(\"src\", \"" + clientAppUrl + "\");\n" +
-        loadAsync +
-        "function loaded(e) {\n" +
+        "var head = document.getElementsByTagName(\"head\")[0] || document.documentElement;\n" +
```

Comment:
never mind, i see it's not available below IE9. with microsoft EOLing IE9 and 10, we'll have to re-evaluate Brisket's browser support in another PR